### PR TITLE
added new policies settings 1909

### PR DESCRIPTION
**Pull request by RAJU2529**
i have added the latest policy settings under the start menu and taskbar. which are available in admx templates 1909.
still, many settings  are there, so I add later once i complete

### CODE REVIEW

**Comment by RAJU2529** [comment]
@illfated @jvsam  just now i added some policy settings from 1909, so please review and approve it if found correct.

currently, I am writing and finding the differences between existing and new policies, once I complete  the work, again i will create PR 
thanking you

**Comment by illfated** [comment]
The syntax & grammar appears to be correct. I might not have time to check up on the technical correctness, so I defer to the document author & manager, @Dansimp , to confirm the correctness of this content.

Rich diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5684/files?short_path=567fe81#diff-567fe8100c6bf5bbcde3018fe42530ca

File: windows/client-management/new-policies-for-windows-10.md [hunk 1]
```diff
@@ -26,6 +26,19 @@ ms.topic: reference
 Windows 10 includes the following new policies for management. [Download the complete set of Administrative Template (.admx) files for Windows 10](https://www.microsoft.com/download/100591).
 
 
+## New Group Policy settings in Windows 10, version 1909
+
+The following Group Policy settings were added in Windows 10, version 1909
```
Comment (illfated) [inline windows/client-management/new-policies-for-windows-10.md]: cosmetic/codestyle: add a colon at the end of the sentence.
```suggestion
The following Group Policy settings were added in Windows 10, version 1909:
```

**Comment by RAJU2529** [comment]
I need update

**Comment by illfated** [comment]
@RAJU2529 : Sorry for not quite getting your point. What kind of update do you mean?

```MarkDown
---
title: New policies for Windows 10 (Windows 10)
description: Windows 10 includes the following new policies for management.
ms.assetid: 1F24ABD8-A57A-45EA-BA54-2DA2238C573D
ms.reviewer: 
manager: dansimp
ms.author: dansimp
keywords: ["MDM", "Group Policy"]
ms.prod: w10
ms.mktglfcycl: manage
ms.sitesec: library
author: dansimp
ms.localizationpriority: medium
ms.date: 10/24/2017
ms.topic: reference
---
```
CC: @Dansimp 

#sign-off

**Comment by e0i** [comment]
@RAJU2529 It might take a while for document authors to get back with a response. Let's have a quick style review go under in the meantime. 

@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by RAJU2529** [comment]
@e0i . I keep waiting

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@lizap Could you take a look at the PR as the co-author of the article in Dan's absence? Thanks.

**Comment by e0i** [comment]
@lizap Please take a look at this community contributed PR as the time permits. Thank you, and stay healthy.

**Comment by ManikaDhiman** [comment]
Hi @RAJU2529, Wanted to give you a quick update on this. I am still awaiting response from the product team, but it seems the **Start Menu and Taskbar** policies that you added for 1909 version were already added in previous versions. I understand that these policies are missing from the doc, but these are not the new policies added in version 1909. I looked into CSP docs and some of these policies are already referenced there for earlier versions. Would it be possible for you to identify the version in which these were added and create a new PR? Thanks for all your help in keeping our docs up to date!

**Comment by RAJU2529** [comment]
@ManikaDhiman . Currently I am using v2004, so now it's not possible to compare 1909 admx templates with v2004, because I don't have another machine to install windows 10 1909 os.
I would like to get  help from 1909 based users to add new policies in this article. Also I request contributers need to look this this article

**Comment by e0i** [comment]
Closing the PR based on Manika's response above. @RAJU2529 Feel free to create a new PR in the light of the author's suggestions. Thank you.

**Pull request closed without merging**